app/src/main/AndroidManifest.xml

```diff
@@ -101,9 +101,10 @@
             android:value=".ui.SearchActivity" />
 
         <!-- Activities -->
-        <activity android:name=".ui.ConversationList"
+        <activity android:name=".ui.ConversationActivity"
             android:launchMode="singleTop"
-            android:theme="@style/Theme.Light">
+            android:theme="@style/Theme.Light"
+            android:configChanges="orientation|screenSize">
```

Comment:
yes: if the contact list is open and the screen rotates, the action bar will be reset to the settings for the conversion list. These configChanges  prevent this by simply ignoring the screen changes.